README.md

```diff
@@ -13,3 +13,26 @@ This guide provides a uniform way to structure your RiotJS code. Making it
 * easier to cache and serve bundles of code separately.
 
 This guide is inspired by [AngularJS Style Guide](https://github.com/johnpapa/angular-styleguide) by John Papa.
+
+## Table of Contents
+
+1. [Component / Tag names](#component-tag-names)
+
+## Component / Tag names
+
+Component or tag names need to be:
+
+* **Spec compliant**: [include a hyphen](https://www.w3.org/TR/custom-elements/#concepts).
+* **Meaningful**: not overspecific, not overly abstract.
+* **Short**: 2 or 3 words.
+* **Pronouncable**: we want to be able talk about them.
+* **`app` namespaced**: using a standard namespace allows easy reuse in other projects
```

Comment:
I feel that prefixing everything with `app-` makes it less meaningful, less pronounceable and obviously longer. So breaking most of the other rules. I think we need to come up with something smarter for this.